### PR TITLE
Make filter by rating preview consistent with default settings

### LINKED ISSUE #9958: Filter by Rating Preview discrepancy
When there are no product ratings in the store, the mock display shows the product count. We want it to not show the product count because in [this PR](https://github.com/woocommerce/woocommerce-blocks/pull/9833) we changed the default state to not show product count. They need to be consistent.

image

### PR DESCRIPTION
**Pull request by roykho**
Fixes https://github.com/woocommerce/woocommerce-blocks/issues/9958

This PR updates the preview to be consistent with default settings.

### Screenshots

| Before | After |
| ------ | ----- |
|        |       |

#### User Facing Testing

1. Ensure you don't have any product reviews on your store.
2. Add a new post and add the `Filter by Rating` block.
3. Because you don't have any reviews, it will show a preview of what the ratings filter would look like.
4. Ensure you don't see any product counts because by default, the `Display Product count` setting is disabled.

* [ ] Do not include in the Testing Notes 

### WooCommerce Visibility

* [x] WooCommerce Core
* [ ] Feature plugin
* [ ] Experimental

### Changelog

> Make filter by rating preview consistent with default settings

### CODE REVIEW

File: assets/js/blocks/rating-filter/preview.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ export const previewOptions = [
 				className={ '' }
```
Comment (imanish003) [inline assets/js/blocks/rating-filter/preview.tsx]: Maybe we can also remove all empty classNames? i.e. `className={ '' }`

**Review by imanish003 — APPROVED** [review]
Hi @roykho, I tested the PR and it's working as expected. 🚀

I left one minor suggestion. Also, I noticed that the UI isn't looking as expected. In the screenshot you can see that the number of faded stars is not correct. Since we are providing ratings from 5 to 1, the second row should have 1 faded star at the end, the third row should have 2 faded stars at the end, and so on.
image

Although I don't think this falls within the scope of the current PR, I am pre-approving it. However, feel free to work on fixing the UI in this PR or we can create a new task for it. 🙂

**Comment by roykho** [comment]
Hi @imanish003 - I've removed the `className` as suggested. As for the stars not showing correctly, that is a side effect of GB. If you deactivate the GB plugin, you will see it correct. As shown below:

**Comment by imanish003** [comment]
Hi @roykho, I just confirmed that disabling Gutenberg fixes the issue. Thank you for explaining it to me. The changes you made look good to me. Well done! 🚀 🚢

**Pull request merged**